oci/pull.bzl

```diff
@@ -218,6 +271,17 @@ Falling back to using `curl`. See https://github.com/bazelbuild/bazel/issues/178
 
     return manifest, len(bytes)
 
+def _create_downloader(rctx):
+    state = {
+        "config": json.decode(rctx.read(rctx.attr.config)),
```

Comment:
should there be some error handling in case you point us to a wrong file or it isn't valid json?

Reply:
json.decode throws in invalid json case. for other cases, we check if the key we are looking for exists in the json file. so no for now.